sdk/openai/Azure.AI.OpenAI/tests/OpenAIImageTests.cs

```diff
@@ -50,6 +50,16 @@ public async Task CanGenerateImages(Service serviceTarget)
             ImageGenerationData firstImageLocation = imageGenerations.Data[0];
             Assert.That(firstImageLocation, Is.Not.Null);
             Assert.That(firstImageLocation.Url, Is.Not.Null.Or.Empty);
+
+            // ContentFilterResults and PromptFilterResults are currently Azure Only
+            if (serviceTarget is Service.Azure)
+            {
+                foreach (ImageGenerationData data in imageGenerations.Data)
+                {
+                    Assert.That(data.ContentFilterResults, Is.Null.Or.Empty);
+                    Assert.That(data.PromptFilterResults, Is.Not.Null.Or.Empty);
```

Comment:
Can we also add code to check the fields inside ContentFilterResults and PromptFilterResults?

Reply:
https://github.com/ninghu/azure-sdk-for-net/pull/16/commits/4181609d4fd660ad3af7bc1115eefbfda6b04645